src/common/imageio_tiff.c

```diff
@@ -259,6 +266,7 @@ static inline int _read_planar_16_Lab(tiff_t *t, uint16_t photometric)
 
   for(uint32_t row = 0; row < t->height; row++)
   {
+    float range = photometric == PHOTOMETRIC_CIELAB ? 65535.0f : 65280.f;
```

Comment:
Add parenthesis for clarity. 
Does it need to be set for each row? Otherwise move out of loop .. compiler will do it anyway.

Reply:
Absolutely right, thanks for the review!